packages/stores/src/mockStore.ts

```diff
@@ -263,6 +264,18 @@ const mock = (): TStores => {
             selectRegion: jest.fn(),
             is_low_risk_cr_eu_real: false,
         },
+        trade: {
```

Comment:
@henry-deriv There is no point in adding partial types for trade-store. Updating this file is out of the scope of this PR, we don't have to add types to the stores, instead we need to add types to the components assuming that the store will be typed later outside of this PR. )

Reply:
oh no. We get CircleCI/Codecov error. It's complaining on almost everyone's PR that involves using types.ts

Reply:
@maryia-deriv if we use useStore and there in no types for your parameters, we should add types in store types and to mock store types as well. this can be a scope of your card if you use useStore and mockStore

Reply:
Actually, it's a good thing if anyone can help in their PRs to add more types to the `packages/stores/types.ts`.

As we know even if (**if** 😬) we migrate all the stores in the core package to TS still we can not infer their types and the `packages/stores/types.ts` file is the only way we can (**manually**) have the types in other packages.

This file is being maintained by all devs and every day devs add more types to it so that eventually we can have all the types for the stores. It's definitely **not pointless**, Please do keep the changes as it helps other devs 🙏🏻 Thanks for the effort @henry-deriv 🌹

Reply:
@henry-deriv @farzin-deriv The scope of the PR is to migrate several components and their tests to TS. There is a reason to limiting the scope of every PR to its purpose. Also, there is no point in partial TS migration of trade-store. A separate PR needs to be created for migration of the entire trade-store at once. @yauheni-deriv useStore is not used in any of the several components that need to be migrated to TS in this PR.  @henry-deriv As I mentioned in other comments here, props types of our components should be independent of any external types, i.e. trade-store fields' types. Otherwise, when trade-store fields change together with their types, TS will never warn us that our component needs to be changed too because our component is built to work with props of specific types.

Reply:
@maryia-deriv There won't be any prop to have a type anymore after removing the connect, Everything will be inferred :(
As I suggested before we can have a call so I can explain the changes we are making 🙇🏻 🌹 

cc: @matin-deriv 

---

@henry-deriv Let's remove the types you added in `packages/stores/types.ts` as @maryia-deriv suggested so we can move forward 🙏🏻